JavaAssignment/src/kunal/PerimeterArea.java

```diff
@@ -0,0 +1,35 @@
+package kunal;
+public class PerimeterArea 
+{
+    int radius, length, breadth;
+    double pi = 3.14;
+	double perimeter;
+	double Area;
+	double Area1;
+   
+    void circle(int radius)
```

Comment:
Give proper method name, not getting what are you going to do with "circle" method. If you are finding an area of circle write like "areaOfCircle" as a method name.